src/Text/LLVM/AST.hs

```diff
@@ -657,6 +657,11 @@ isIArith _      = False
 isFArith :: ArithOp -> Bool
 isFArith  = not . isIArith
 
+data UnaryArithOp
```

Comment:
This is intended to mirror the existing `ArithOp` instruction. One could reasonably argue that `ArithOp` should now be called `BinaryArithOp` (and similarly, `Arith` should be called `BinaryArith`), but I opted not to do that so as to avoid a breaking API change. Let me know if you'd prefer to be bolder and renaming things for the sake of consistency.